src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2.java

```diff
@@ -132,23 +125,12 @@ public void onPrimary() {
 		modalPresenter.setLoading(true);
 		accessRequirement.setAreOtherAttachmentsRequired(view.areOtherAttachmentsRequired());
 		String expirationPeriod = view.getExpirationPeriod();
-		String renewalDetailsURL = view.getRenewalDetailsURL();
-		
-		if (!ValidationUtils.isValidUrl(renewalDetailsURL, true)) {
-			modalPresenter.setErrorMessage(INVALID_RENEWAL_URL_MESSAGE);
```

Comment:
You might want to delete the now unused INVALID_RENEWAL_URL_MESSAGE constant

Reply:
Done